pkg/printers/printers.go

```diff
@@ -59,7 +59,7 @@ func GetStandardPrinter(mapper meta.RESTMapper, typer runtime.ObjectTyper, encod
 		templatePrinter.AllowMissingKeys(allowMissingTemplateKeys)
 		printer = templatePrinter
 
-	case "templatefile", "go-template-file":
+	case "go-template-file":
```

Comment:
Why is this being removed?  If it's possible for a user to specify this, this would not be a backwards compatible change on the CLI.

Reply:
I remove it because I didn't saw this output option in `kubectl get -h` or anywhere, but if for backwards compatible, that's make sense.
Sorry I'm not familiar with the early version.

Reply:
Fixed, @smarterclayton  : )